backend/uv-backend.py

```diff
@@ -0,0 +1,437 @@
+                        order_detail['commentLabel']=(details['TRANSACTION'][x]['SHIP.BOLNUMBER'])
+            void_reason=details['TRANSACTION'][0]['SHIP.ADDR_MV'][0]['SHIP.ADDR']
+            if(void_reason):
+                if('AUDIT.FLAG' in details['TRANSACTION'][0].keys()):
+                    audit_flag_record=details['TRANSACTION'][x]['AUDIT.FLAG']
+                    order_detail['auditFlag']='AUDIT FLAG: '+audit_flag
+                    return_count=details['TRANSACTION'][0]['RETURN.TRANS_MV'][0]['RETURN.TRANS']
+                    if (audit_flag_record.count("Return ")>1 or audit_flag_record.count("XCHG ")>1):
+                        audit_flag="RETURN REASON: "+str(audit_flag)
+                    else:
+                        audit_flag='VOID REASON: '+audit_flag
+            coupons=details['TRANSACTION'][0]['GEN.COUPON.ID_MV'][0]['GEN.COUPON.ID']
+            if(coupons):
+                order_detail['coupons']='GENERATED COUPON(S): '+coupons
+            if('AUDIT.FLAG' in details['TRANSACTION'][0].keys()and details['TRANSACTION'][0]['AUDIT.FLAG']):
+                order_detail['auditFlag']='AUDIT FLAG: '+audit_flag
+                return_count=details['TRANSACTION'][0]['RETURN.TRANS_MV'][0]['RETURN.TRANS']
+                for i in range(len(return_count)):
+                    order_detail['returnIds']=details['TRANSACTION'][0]['SHIP.ADDR_MV'][x]['SHIP.ADDR']
+            if('ERCPT.EMAIL' in details['TRANSACTION'][0].keys()):
+            	email=details['TRANSACTION'][0]['ERCPT.EMAIL']
+            	order_detail['email']='ERCPT TO: '+email
+            response={
+                "orderDetail": order_detail,
+                    }
+            return Response(
+                json.dumps(response),
+                status=200,
+                mimetype='application/json'
+                
+            )
+        else:
+            msg='Order number not found'
+            data={
+                 'msg':msg        
+                 }
+            return Response(
+                json.dumps(data),
+                status=404,
+                mimetype='application/json'
+            )
+    else:
+        msg ='Transaction file not found'
+        data={
+             'msg':msg
+             }
+        return Response(
+            json.dumps(data),
+            status=404,
+            mimetype='application/json'
+        )
+
+#############################################################
+###################### Discount API #########################
+#############################################################
+        
+@app.route('/api/transaction/discount/<transactionId>', methods=['GET'])
+def discount(transactionId):
+    transaction_id= transactionId
+    status = check_existing_record('TRANSACTION',transaction_id )
+    if(status):
+        #data_file = u2py.File("TRANSACTION")
+        cmd="LIST DATA RESERVATIONS DISCOUNT.TYPE ".format()
+        details=u2py.Command(cmd).run(capture=True)
```

Comment:
Are we even making use of `transaction_id`?

It seems like this would return all records irrespective of what `transaction_id` is passed